scripts/commitlint.sh

```diff
@@ -7,9 +7,11 @@ set -o errexit
 if [ -n "${CIRCLE_PULL_REQUEST}" ]
 then
   PULL_REQUEST_DETAILS=$(curl https://api.github.com/repos/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}/pulls/${CIRCLE_PR_NUMBER})
-  BASE_SHA1=$(echo "${PULL_REQUEST_DETAILS}" | jq -r .base.sha)
-  HEAD_SHA1=$(echo "${PULL_REQUEST_DETAILS}" | jq -r .head.sha)
+  BASE_SHA1=$(node -pe 'JSON.parse(process.argv[1]).base.sha' "${PULL_REQUEST_DETAILS}")
+  echo "Base SHA1: $BASE_SHA1"
```

Comment:
extra `echo`'s?

Reply:
Just for some info to be listed while it runs

Reply:
Ok just wanted to check because it should be printed out on L15 as part of the `COMMIT_RANGE` too right?